spec/controllers/chapters_controller_spec.rb

```diff
@@ -2,33 +2,33 @@
 
 describe ChaptersController do
   context "routing" do
-    let(:boston){ FactoryGirl.build_stubbed(:chapter) }
-
     it "routes /chapters/boston to chapters#show" do
-      {:get => "/chapters/boston"}.should
-        route_to({:controller => "chapters", :action => "show", :id => boston.id, :locale => "en"})
+      expect({:get => "/chapters/boston"}).to route_to(
+        {:controller => "chapters", :action => "show", :id => "boston"}
```

Comment:
To get this working, I had to fix the expected route -- it had fallen out of sync at some point.